tests/ui/spirv-attr/invalid-target.rs

```diff
@@ -30,6 +30,8 @@
 //   * 6 on `_entry_param`
 //   * 1 on `_closure`
```

Comment:
Okay good, this accounting (expand to see it above, these are just the last 2 lines of a dozen) is still correct most likely, total error count is still 473.